### PR TITLE
docs: render image-scanner PlantUML diagram

### PR DESCRIPTION
**Pull request by mikaelol**
Fixes https://github.com/statnett/image-scanner-operator/issues/4 (using the method described by https://github.com/statnett/image-scanner-operator/pull/26#issuecomment-1377531957).

Example using the file in this branch: <http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mikaelol/image-scanner-operator/render-plantuml-architecture/docs/architecture.puml>

### CODE REVIEW

File: architecture.puml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@startuml Image Scanner architecture
```
Comment (erikgb) [inline architecture.puml:1]: Should we move this file into a docs folder?

**Review by erikgb — COMMENTED** [review]
Nice!!!! Just a couple of comments/questions.

File: architecture.puml (hunk 1 above)
Comment (mikaelol) [reply architecture.puml:1]: Sounds like a good idea, fixed now.

**Review by erikgb — APPROVED** [review]
LGTM, is the URL targeting the main branch now? Possible to make the URL more human-readable?

**Comment by mikaelol** [comment]
> LGTM, is the URL targeting the main branch now? Possible to make the URL more human-readable?

Yes, [the URL ](https://www.plantuml.com/plantuml/png/5Ssn3G8n30NGdYbWW8jxgbl5EDObKk94zhTO7wvwvJi3xCWv7gzkCh9g-dWsOCURoFbRhewMdmnrMGOr55cJ0WnJWFhaIuyGDbC_rbPdBAV9tQWk2MANrg62T2txdawr-lS7) is targeting `main`, so it does not work now since there is no file present there.

This is the URL to the editor: http://www.plantuml.com/plantuml/umla/JOunhWCn24JxVugV0wpfK-rL24PhAnWi3Chru-rIZMQaUNhd2lJ88N-FhYHPELt-MyHSJm37Jxbwj7pbOYVJO8r2Dc3qDxAOG1zuyR48LTaFc-mOvk2iTVSfg3VzGAVMWodI6QhHWj-bpDlWt9-TNm00 , and I can't see any option there to make the URL more readable.

**Comment by mikaelol** [comment]
Hm, I am not sure this was such a good idea as described in https://github.com/statnett/image-scanner-operator/issues/4#issuecomment-1373743296 after all. It seems like the image/PlantUML code is cached, so even if the raw PlantUML source code is changed, the diagram does not seem to change.

Sorry for not testing this more thoroughly before creating the PR.

**Comment by erikgb** [comment]
Can we try this: https://github.com/jonashackt/plantuml-markdown#2-integrate-plantuml-render-engine-with-github-markdown? It makes the URL a lot more readable also.

**Comment by mikaelol** [comment]
> Can we try this: https://github.com/jonashackt/plantuml-markdown#2-integrate-plantuml-render-engine-with-github-markdown? It makes the URL a lot more readable also.

Cool, this seems to be working much better! Updated the description and content of this PR to use this approach now.

**Review by erikgb — APPROVED** [review]
Nice! Looks good!

**Pull request merged**